### DISCUSSION TITLE
The Levitating Emojis Aren't In The Style Of Anything Else.

### DISCUSSION THREAD

**Discussion by Bronnel721** [C1710/blobmoji#109] [Style]
![Screenshot (70)](https://user-images.githubusercontent.com/92538982/159139177-d28518a8-f515-4d2e-b89c-978845da3309.png)
The only one that's in the style of everything else is this:🕴️.

**Comment by C1710** [comment]
✔️ 054b4629ff3b52120c23b41d049325dbd3cc9394
